cli/medperf/commands/dataset/prepare.py

```diff
@@ -309,8 +309,13 @@ def prompt_for_report_sending_approval(self):
             + " dataset subjects have reached Stage 1, and that 60% of your dataset subjects"
             + " have reached Stage 3:"
         )
+        warning = (
+            "Note that reports will be visible by the Data Preparation MLCube owner and by the"
+            " Benchmark owner to keep track of progress and provide support if needed."
+        )
         config.ui.print(msg)
         dict_pretty_print(example)
+        config.ui.print_warning(warning)
```

Comment:
We should have a message during dataset submission notifying the user (in a warning color maybe) that their user information will be visible to the owner of the preparation mlcube they specified (we should explicitly mention what will be visible)

Reply:
i.e., during "dataset submission", not here. when we ask the user for confirmation of submission, in the same prompt we write a yellow text stating that "your email address will be visible to the data preparation MLCube owner".